ZAPD/ZFile.cpp

```diff
@@ -1095,41 +880,73 @@ string ZFile::ProcessDeclarations()
+			//output += StringHelper::Sprintf("%s %s[] = {\n#include \"%s\"\n};\n\n", item.second->varType.c_str(), item.second->varName.c_str(), item.second->includePath.c_str());
+
+			if (item.second->isExternal)
+			{
+				// HACK
+				std::string extType = "";
+
+				if (item.second->varType == "Gfx")
+					extType = "dlist";
+				else if (item.second->varType == "Vtx" || item.second->varType == "static Vtx")
+					extType = "vtx";
+
+				File::WriteAllText(StringHelper::Sprintf("%s/%s.%s.inc", Globals::Instance->outputPath.c_str(), item.second->varName.c_str(), extType.c_str()), item.second->text);
+			}
+
+			/*if (item.second->varType == "u64")
+				output += StringHelper::Sprintf("#pragma INC_ASSET_U64(\"%s\", \"%s\")\n", item.second->varName.c_str(), item.second->includePath.c_str());
+			else if (item.second->varType == "Gfx")
+				output += StringHelper::Sprintf("#pragma INC_ASSET_GFX(\"%s\", \"%s\")\n", item.second->varName.c_str(), item.second->includePath.c_str());
+			else if (item.second->varType == "Vtx" || item.second->varType == "static Vtx")
+				output += StringHelper::Sprintf("#pragma INC_ASSET_VTX(\"%s\", \"%s\")\n", item.second->varName.c_str(), item.second->includePath.c_str());
+			else
+				output += StringHelper::Sprintf("#pragma INC_ASSET_U8(\"%s\", \"%s\")\n", item.second->varName.c_str(), item.second->includePath.c_str());*/
+
+			// Do not asm_process vertex arrays. They have no practical use being overridden.
+			//if (item.second->varType == "Vtx" || item.second->varType == "static Vtx")
+			if (item.second->varType != "u64" && item.second->varType != "static u64" && item.second->varType != "u8" && item.second->varType != "static u8")
+			{
+				//output += StringHelper::Sprintf("%s %s[] = {\n    #include \"%s\"\n};\n\n", item.second->varType.c_str(), item.second->varName.c_str(), StringHelper::Replace(item.second->includePath, "assets/", "../assets/extracted/").c_str());
+				output += StringHelper::Sprintf("%s %s[] = {\n    #include \"%s\"\n};\n\n", item.second->varType.c_str(), item.second->varName.c_str(), StringHelper::Replace(item.second->includePath, "assets/", "../assets/").c_str());
+				//output += StringHelper::Sprintf("%s %s[] = {\n    #include \"%s\"\n};\n\n", item.second->varType.c_str(), item.second->varName.c_str(), Path::GetFileName(item.second->includePath).c_str());
+			}
+			else
+			{
+				//output += StringHelper::Sprintf("%s %s[] = {\n    #pragma INC_ASSET(\"%s\")\n};\n\n", item.second->varType.c_str(), item.second->varName.c_str(), item.second->includePath.c_str());
+				//output += StringHelper::Sprintf("%s %s[] = {\n    #include \"%s\"\n};\n\n", item.second->varType.c_str(), item.second->varName.c_str(), StringHelper::Replace(item.second->includePath, "assets/", "assets/extracted/").c_str());
+				output += StringHelper::Sprintf("%s %s[] = {\n    #include \"%s\"\n};\n\n", item.second->varType.c_str(), item.second->varName.c_str(), item.second->includePath.c_str());
+			}
```

Comment:
It may be because I'm sleepy, but I don't get what you are trying to do here. 
Process the resources that were written to external files?